scene/main/node.h

```diff
@@ -253,6 +253,11 @@ class Node : public Object {
 		NOTIFICATION_APPLICATION_FOCUS_IN = MainLoop::NOTIFICATION_APPLICATION_FOCUS_IN,
 		NOTIFICATION_APPLICATION_FOCUS_OUT = MainLoop::NOTIFICATION_APPLICATION_FOCUS_OUT,
 		NOTIFICATION_TEXT_SERVER_CHANGED = MainLoop::NOTIFICATION_TEXT_SERVER_CHANGED,
+
+		// Editor specific node notifications
+		NOTIFICATION_EDITOR_PRE_SAVE = 9001,
+		NOTIFICATION_EDITOR_POST_SAVE = 9002
```

Comment:
Should have trailing comma to avoid diff next time we add more values.

What's the rationale for using 9001 and 9002 specifically?

Reply:
The rationale was to better distinguish it from the other notifications, which can occur in game. The editor saving notifications will only occur in the editor and therefore I wanted to make it distinct at a glance from both value and name.